mkdocs/themes/mkdocs/base.html

```diff
@@ -164,6 +168,37 @@
                             </li>
                         {%- endif %}
                       {%- endblock %}
+                      {%- if config.theme.user_color_mode_toggle %}
+                            <li class="nav-item dropdown" data-bs-theme="{% if config.theme.nav_style == 'dark' %}dark{% else %}light{% endif %}">
+                              <a class="nav-link dropdown-toggle d-flex align-items-center" href="#" id="theme-menu" aria-expanded="false" data-bs-toggle="dropdown" data-bs-display="static" aria-label="Toggle theme">
+                                <i class="fa-solid fa-circle-half-stroke fa-fw"></i>
+                                <span class="d-lg-none ms-2">Toggle theme</span>
+                              </a>
```

Comment:
The dropdown is off-center vertically compared to other nav items. Let's drop `d-flex` from it, that's what causes it. Also change to button.

```suggestion
                              <button id="theme-menu" aria-expanded="false" data-bs-toggle="dropdown" data-bs-display="static" aria-label="Toggle theme" class="nav-link dropdown-toggle">
                                <i class="fa-solid fa-circle-half-stroke fa-fw"></i>
                                <span class="d-lg-none ms-2">Toggle theme</span>
                              </button>
```

Reply:
Ah, I noticed that off-center, but couldn't figure out what was causing it. So thanks.

Regarding changing to `button`, why? All the other nav items use `a` links.

Reply:
Why button: because unlike other items, it's just a button that doesn't lead anywhere :)
Good for semantic browsing / accessibility.

Applied this suggestion in https://github.com/mkdocs/mkdocs/commit/a83615554940f681b9c292b21de78e84616017c8